Makefile

```diff
@@ -25,18 +30,10 @@ BUILD_BATS_SUPPORT_VERSION ?= 004e707638eedd62e0481e8cdc9223ad471f12ee
 DOCKER_IMAGE_BATS := cftk/bats
 # DOCKER_TAG_BATS is the image semver and has no correlation to bats versions
 DOCKER_TAG_BATS ?= 0.6.0
-# Integration Testing Versions
-DOCKER_IMAGE_INTEGRATION := cft/kitchen-terraform
-# These versions should match the same versions used in the Pipeline definition
-BUILD_TERRAFORM_VERSION ?= 0.11.10
-BUILD_CLOUD_SDK_VERSION ?= 216.0.0
-BUILD_PROVIDER_GOOGLE_VERSION ?= 1.19.1
-BUILD_PROVIDER_GSUITE_VERSION ?= 0.1.10
-DOCKER_IMAGE_INTEGRATION_TAG ?= ${BUILD_TERRAFORM_VERSION}_${BUILD_CLOUD_SDK_VERSION}_${BUILD_PROVIDER_GOOGLE_VERSION}_${BUILD_PROVIDER_GSUITE_VERSION}
-DOCKER_IMAGE_INTEGRATION_URI ?= gcr.io/cloud-foundation-cicd/${DOCKER_IMAGE_INTEGRATION}:${DOCKER_IMAGE_INTEGRATION_TAG}
+
 
 # All is the first target in the file so it will get picked up when you just run 'make' on its own
-all: check_shell check_python check_golang check_terraform check_docker check_base_files test_check_headers check_headers check_trailing_whitespace generate_docs
+all: check_terraform check_shell check_python check_golang check_docker check_base_files test_check_headers check_headers check_trailing_whitespace generate_docs
```

Comment:
Please remove check_docker, it deprecated due to standard containers usage.

Reply:
we still have unique container with bats support here, until it will be replaced with developer-tools I would keep it